src/body.js

```diff
@@ -1,12 +1,12 @@
 'use strict'
 
 const { kBody, kBodyUsed } = require('./symbols')
-const { isReadable } = require('./utils')
+const { isAsyncIterable } = require('./utils')
 
 class Body {
   constructor (input = null) {
-    if (input != null && !isReadable(input)) {
-      throw Error('body must be undefined, null, or a readable stream')
```

Comment:
I think sync iterable is also fine?

Reply:
Potentially, have to dig into the idl definition of `async iterator` from the whatwg readablestream defintion (https://streams.spec.whatwg.org/#rs-class-definition)

Reply:
i'm not great at understanding idl definition, but I do not believe an async iterable is also an iterable

Reply:
No. But an iterable is also an async iterable.

Reply:
At least in terms of iteration.

Reply:
oh yes. and this would allow folks to pass strings directly to the `body` property

Reply:
Taking a quick peak under the covers. in order to be actually spec compliant its a bit more complicated: https://github.com/node-fetch/node-fetch/blob/1780f5ae89107ded4f232f43219ab0e548b0647c/src/body.js#L35

Reply:
for now I'm supporting iterables out of the box. when I dive into spec compliant work on Body i'll fix up the input argument coercion